lib/cfa/augeas_parser.rb

```diff
@@ -101,12 +112,28 @@ class AugeasTree
     # Low level access to Augeas structure
     #
     # An ordered mapping, represented by an Array of Hashes
-    # with the keys :key and :value.
+    # with the keys :key, :value, :operation and :orig_key.
+    #
+    # - `:key` is modified key without index used by collection
+    # - `:value` is string value or instance of AugeasTree or AugeasTreeValue
+    # - `:operation` is internal variable holding modification of augeas
+    #   structure. It is used for minimal modification of source file.
```

Comment:
and :keep. But it is internal variable. So it is intention as it is internal variable and you should not rely on it, only know to ignore it.